tests/src/Kernel/MenuLocalTasksTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel;
+
+use Drupal\Core\Url;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests that Drupal local tasks are properly rendered.
+ */
+class MenuLocalTasksTest extends AbstractKernelTestBase {
+
+  /**
+   * Test menu local tasks.
+   */
+  public function testMenuLocalTasks(): void {
+    $render = [
+      '#theme' => 'menu_local_tasks',
+      '#primary' => [
+        'link1.link' => [
+          '#theme' => 'menu_local_task',
+          '#link' => [
+            'title' => 'Third link - Active',
+            'url' => Url::fromUri('http://www.active.com'),
+          ],
+          '#active' => TRUE,
+          '#weight' => 10,
+        ],
+        'link2.link' => [
+          '#theme' => 'menu_local_task',
+          '#link' => [
+            'title' => 'First link - Inactive',
+            'url' => Url::fromUri('http://www.inactive.com'),
+          ],
+          '#active' => FALSE,
+          '#weight' => -10,
+        ],
+        'link3.link' => [
+          '#theme' => 'menu_local_task',
+          '#link' => [
+            'title' => 'Second link',
+            'url' => Url::fromUri('http://www.middlelink.com'),
+          ],
+          '#active' => FALSE,
+          '#weight' => 0,
+        ],
+      ],
+    ];
+
+    $html = $this->renderRoot($render);
+    $crawler = new Crawler($html);
+
+    $actual = $crawler->filter('nav.nav-tabs');
```

Comment:
Please, change name $actual to $nav or $navigation. It makes easier to understand at first glance.